app/views/users/new.html.erb

```diff
@@ -0,0 +1,52 @@
+<h2>Create User</h2>
+
+<%= form_for @user, url: {action: "create"} do |f| %>
+  <%= f.label "Username" %>
+  <%= f.text_field :username %>
+  <%= "Username #{@user.errors.messages[:username].first}" if !@user.errors.messages[:username].empty? %><br><br>
+  <%= f.label "Email" %>
+  <%= f.email_field :email, :placeholder => "email@something.dot" %>
+  <%= "Email #{@user.errors.messages[:email].first}" if !@user.errors.messages[:email].empty? %><br><br>
+  
+  <%= f.label "Password" %>
+  <%= f.password_field :password %>
+  <%= "Password #{@user.errors.messages[:password].first}" if !@user.errors.messages[:password].empty? %><br><br>
+  <%= f.submit "Create" %>
+<% end %>
+
+<!--
+<%= form_tag("/users", method: "post") do %>
+  <%= label_tag(:username, "Name:") %>
+  <%= text_field_tag(:username) %> <br><br>
+    <%= label_tag(:email, "Email:") %>
+  <%= email_field_tag(:email) %> <br><br>
+    <%= label_tag(:password, "Password:") %>
+  <%= password_field_tag(:password) %> <br><br>
```

Comment:
- [ ] For each of the label tag here please change it to this format:  `<%= email_field_tag 'email' %>`